prow/scripts/cluster-integration/kyma-gke-long-lasting.sh

```diff
@@ -197,11 +196,29 @@ function generateAndExportLetsEncryptCert() {
         --server https://acme-v02.api.letsencrypt.org/directory \
         --dns-google-propagation-seconds=600 \
         -d "*.${DOMAIN}"
-
+	
     TLS_CERT=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/fullchain.pem | tr -d '\n')
     export TLS_CERT
     TLS_KEY=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/privkey.pem   | tr -d '\n')
     export TLS_KEY
+	#encrypt the tls cert
+	gcloud kms encrypt --location global \
+	--keyring "${KYMA_KEYRING}" \
+	--key "${KYMA_ENCRYPTION_KEY}"
+	--plaintext-file ./letsencrypt/live/"${DOMAIN}"/fullchain.pem  \
+	--ciphertext-file "nightly-gke-tls-integration-app-client-cert.encrypted"
```

Comment:
we should have this implemented once, in a helper bash file. Now we have code duplication.

Reply:
updated in both gke and aks